data/AkshitaKumawat.json

```diff
@@ -0,0 +1,21 @@
+
+{
+  "name": "Akshita Kumawat",
+  "type": "personal",
+  "bio": "OPEN SOURCE COMMUNITY",
```

Comment:
Great that you belong to the Open Source Community - but Bio should be a few words to describe you personally.  

Take a look at a few Profiles on for inspiration: https://linkfree.io

As this is not blocker I will merge this Pull Request.  If you would like to make changes to your Profile then please raise a new Pull Request.  I would suggest you take a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing